code/game/objects/items/devices/megaphone.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "A device used to project your voice. Loudly."
 	icon_state = "megaphone"
 	item_state = "radio"
-	w_class = 1
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
Welp thats true.

Although, why is a Megaphone classed as TINY in the first place... 

To keep this a re-factor I'll change this, but probably just be larger than TINY... as TINY is defined as "Usually items smaller then a human hand, ex: Playing Cards, Lighter, Scalpel, Coins/Money"